File: JeffSebastian/playground/src/main/java/nl/uva/se/ql/ast/expression/logical/And.java
package nl.uva.se.ql.ast.expression.logical;

import nl.uva.se.ql.ast.expression.Binary;
import nl.uva.se.ql.ast.expression.Expression;
import nl.uva.se.ql.ast.expression.ExpressionVisitor;

public class And extends Binary {

	public And(int lineNumber, int offset, Expression left, 
			Expression right) {
		super(lineNumber, offset, left, right);		
	}

	@Override
	public <T> T accept(ExpressionVisitor<T> visitor) {
		return visitor.visit(this);
	}
	
}

File: FelipezConde/QuestionnaireLanguage/AST/Nodes/Expressions/Binary/And.cs
using Expressions = AST.Nodes.Interfaces;

namespace AST.Nodes.Expressions.Binary
{
    public class And : BaseBinary
    {
        public And(BaseExpression left, BaseExpression right, PositionInText position)
            : base(left, right, position)
        {}

        public override string ToString()
        {
            return "&&";
        }

        public override T Accept<T>(ASTVisitors.Interfaces.IExpressionVisitor<T> visitor)
        {
            return visitor.Visit(this);
        }
    }
}
